snapshotter_cli/commands/shell.py

```diff
@@ -340,17 +340,45 @@ def run_shell(app: typer.Typer, parent_ctx: typer.Context):
         )
     )
 
-    # Show latest changes if available
-    latest_changes = get_latest_changes()
-    if latest_changes:
-        console.print(
-            Panel(
-                latest_changes,
-                title="[bold blue]What's New[/bold blue]",
-                border_style="blue",
-                padding=(1, 2),
+    # Show latest changes if available (only once per version)
+    from snapshotter_cli import get_version_string
```

Comment:
Let’s avoid placing imports in the middle of the files. It makes the code harder to read and maintain